### PR TITLE
fix typo in NOTES.txt

### PR DESCRIPTION
**Pull request by slumbi**
Load generator ui link was not correct

### CODE REVIEW

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
@slumbi thanks for the contribution. Please sign the CLA

**Review by TylerHelmuth — COMMENTED** [review]
@slumbi thanks for the contribution.  Please rebase, bump the chart's patch version again, and then run `make generate-examples`

**Review by TylerHelmuth — APPROVED** [review]
@slumbi please run `make generate-examples`

File: charts/opentelemetry-demo/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 type: application
 name: opentelemetry-demo
-version: 0.12.0
+version: 0.12.1
```
Comment (TylerHelmuth) [inline charts/opentelemetry-demo/Chart.yaml:4]: Just merged in another PR, gotta bump this to `0.12.2` and then run `make generate-examples`.

Comment (slumbi) [reply charts/opentelemetry-demo/Chart.yaml]: Ok , done, thank you . But I find this process a bit complex. Shouldn't we bump versions  on every release, and not on every pull  request?  Contributors  need to rebase constantly  just to fix version numbers.
Me I have contributed one character change in the documentation , but  the overhead of making it to master is quite high

Comment (TylerHelmuth) [reply charts/opentelemetry-demo/Chart.yaml]: The process has become quite cumbersome as the charts have become more active. In the past, when there wasn't as much going on, it was a convenient way to ensure the latest charts were always released. 

Can you open an issue on this topic?

**Pull request merged**